packages/adapter-node/src/index.js

```diff
@@ -6,10 +6,55 @@ export const path = env('SOCKET_PATH', false);
 export const host = env('HOST', '0.0.0.0');
 export const port = env('PORT', !path && '3000');
 
+const listen_pid = parseInt(env('LISTEN_PID', '-1'));
+const listen_fds = parseInt(env('LISTEN_FDS', '-1'));
+const timeout = parseInt(env('TIMEOUT', '-1'));
+const systemd_socket_fd = 3;
+
 const server = polka().use(handler);
 
-server.listen({ path, host, port }, () => {
-	console.log(`Listening on ${path ? path : host + ':' + port}`);
-});
+function close() {
+	server.server.closeIdleConnections();
```

Comment:
getting a red squiggly here:

> error TS2339: Property 'closeIdleConnections' does not exist on type 'Server'.

any ideas? do we just need to slap a `// @ts-expect-error` on it, or is it telling us that we can't rely on this existing?

Reply:
Yeah, I'm not sure myself either tbh because [`closeIdleConnections` was added in Node.js 18.2.0](https://nodejs.org/api/http.html#servercloseidleconnections) and `adapter-node` has `"@types/node": "^18.19.3",` in its `package.json`. So we can probably just ignore this.

Reply:
```suggestion
	// @ts-expect-error
	server.server.closeIdleConnections();
```